app/workers/database_size_control.rb

```diff
@@ -0,0 +1,27 @@
+
+module Workers
+  class DatabaseSizeControl < Base
+    include Sidetiq::Schedulable
+
+    sidekiq_options queue: :maintenance
+
+    recurrence { daily }
+
+    def perform
+      if AppConfig.settings.maintenance.database_size_control.enable?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.